src/package.xml

```diff
@@ -2324,9 +2327,23 @@
         <name>EmailTemplate</name>
     </types>
     <types>
+        <members>HasUserManagedTDTM</members>
+        <members>IsEnabled_AddressVerification</members>
+        <members>IsEnabled_AutomaticPayments</members>
         <members>IsEnabled_CustomizableRollups</members>
+        <members>IsEnabled_DefaultGAU</members>
+        <members>IsEnabled_HouseholdAcctModel</members>
+        <members>Org_IsEnabled_AdvancedMultiCurrency</members>
         <name>FeatureParameterBoolean</name>
     </types>
+    <types>
+        <members>Data_Count_of_ErrorLog</members>
```

Comment:
What is the naming convention to use `Data_` or `Org_` for the parameter?
Suggestion, parameters can be read the same or better if not using underscores. For example:
```
<members>ErrorLogCount</members>
<members>AllRecurringDonationOppCount</members>
<members>OpenEndedRecurringDonationOppCount</members>
<members>MaxNumRelatedOpps</members>
<members>Org_NonEnglishLanguageCount</members>
```

Reply:
Can we remove underscores from the names?

Reply:
We're already committed to `IsEnabled_CustomizableRollups` as a FeatureParameter name, so at this point I think the underscores are simply part of the naming pattern. I'd prefer not to have one with an underscore and the rest without.